pq2json/src/converter.rs

```diff
@@ -92,37 +92,21 @@ fn projected_schema(
     )
 }
 
-macro_rules! element_to_value {
-    ($ft:expr, $obj:ident, $i:ident, $settings:ident) => {
-        match $ft {
-            FieldType::Null => Value::Null,
-            FieldType::Bool => Value::Bool($obj.get_bool($i)?),
-            FieldType::Byte => Value::Number($obj.get_byte($i)?.into()),
-            FieldType::Short => Value::Number($obj.get_short($i)?.into()),
-            FieldType::Int => Value::Number($obj.get_int($i)?.into()),
-            FieldType::Long => Value::Number($obj.get_long($i)?.into()),
-            FieldType::UByte => Value::Number($obj.get_ubyte($i)?.into()),
-            FieldType::UShort => Value::Number($obj.get_ushort($i)?.into()),
-            FieldType::UInt => Value::Number($obj.get_uint($i)?.into()),
-            FieldType::ULong => ulong_to_value($obj.get_ulong($i)?, &$settings),
-            FieldType::Float => float_to_value($obj.get_float($i)? as f64),
-            FieldType::Double => float_to_value($obj.get_double($i)?),
-            FieldType::Decimal => Value::String(decimal_to_string($obj.get_decimal($i)?)),
-            FieldType::Str => Value::String($obj.get_string($i)?.to_string()),
-            FieldType::Bytes => bytes_to_value($obj.get_bytes($i)?.data()),
-            FieldType::Date => date_to_value($obj.get_date($i)?)?,
-            FieldType::TimestampMillis => {
-                timestamp_to_value($settings, $obj.get_timestamp_millis($i)?)?
-            }
-            FieldType::TimestampMicros => timestamp_to_value(
-                $settings,
-                $obj.get_timestamp_micros($i).map(|ts| ts / 1000)?,
-            )?,
-            FieldType::Group => row_to_value($settings, $obj.get_group($i)?)?,
-            FieldType::List => list_to_value($settings, $obj.get_list($i)?)?,
-            FieldType::Map => map_to_value($settings, $obj.get_map($i)?)?,
-        }
-    };
+fn element_to_value(settings: &Settings, field: &Field) -> Value {
```

Comment:
I don't remember what were the changes made by Evgeney in his fork of Apache Arrow. Are we sure we don't miss these changes in the latest official Apache Arrow version?